util/devel/test/vagrant/provision-scripts/chapel-setmakej.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-MAKEJ=${MAKEJ:=-j 4}
+MAKEJ=${MAKEJ:=-j 16}
```

Comment:
Nice!
Will try to use this in our docker builds.

Reply:
Ideally it would depend on the system's memory and the number of cores. This setting works on my own system where I run this and I would expect it to work on chapdl etc.